deployment

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+sudo apt update
+
+sudo apt install python3.8
+
+
+sudo apt install python3-pip
+
+sudo apt install python3-venv
+
+cp webapp.py /var/webapp
+
+mkdir project
+
+cp webapp.py project
+
+cd project
+
+python3 -m venv venv
+
+source venv/bin/activate
+
+pip install Flask
+
+python -m flask --version
+
+export FLASK_APP=webapp.py
+
+flask run
```

Comment:
I see in the output     `default:  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)`
But in fact when trying to access this port I get this error:
ERR_CONNECTION_RESET 
are you able to see the 'Hello world!' web page?

Reply:
@georgef21 - Yonatan has a point here. Please research.